docs/installation.md

```diff
@@ -64,6 +64,9 @@ m.render(document.body, "hello world");
 ```bash
 $> npm start
 ```
+
+**Note:** If you see an error `Error: Can't resolve 'bin/app.js'`, go ahead and create that file.
```

Comment:
I'm confused by this note. The webpack entry in `package.json` combined with `npm start` should happily create that file. Did it not for you?

Reply:
It didn't, and I had a couple friends try and it also didn't. It definitely should, I agree with you, but following these steps exactly, it did not across 3 separate computers.

Reply:
Ok, instead of band-aiding with a docs comment I'm going to investigate why webpack isn't creating that file correctly. Would you be willing to open an issue on that particular problem and remove the line from this PR for now? We can always bring it back if I can't figure out what the issue is :)

Reply:
Ah, looks like webpack's CLI usage has changed since this was written. I'll fix it up. Thanks again for the PR!

Reply:
Ok webpack usage fix up as #2094, remove the line from this PR and we're good to :ship: